core/src/main/resources/css/atoms/_audio.scss

```diff
@@ -69,24 +69,19 @@
 
 /* CSS STYLES FOR AUDIO ATOM  - convert to Sass */
```

Comment:
Question, why the note "convert to Sass" ?

Reply:
🤔 no idea. I'll remove!

Reply:
It is a note to change from CSS to SCSS - ie use nesting etc to clear up the code

Reply:
Ah I see! In future, best to label with a `TODO`. I was thinking "but it is an scss file so it will definitely be converted to css".